docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
-| statistic_analyze_status_keep_second     | s    | 259200       | The duration to retain the history of collection tasks. The default value is 3 days. Unit: seconds. |
-|statistic_collect_concurrency             | -    |  3  | The maximum number of manual collection tasks that can run in parallel. The value defaults to 3, which means you can run a maximum of three manual collections tasks in parallel. <br>If the value is exceeded, incoming tasks will be in the PENDING state, waiting to be scheduled.|
-| enable_local_replica_selection           | -    | FALSE        | Whether to select local replicas for queries. Local replicas reduce the network transmission cost. If this parameter is set to TRUE, the CBO preferentially selects tablet replicas on BEs that have the same IP address as the current FE. If this parameter is set to FALSE, both local replicas and non-local replicas can be selected. The default value is FALSE. |
-| max_distribution_pruner_recursion_depth  | -    | 100          | The maximum recursion depth allowed by the partition pruner. Increasing the recursion depth can prune more elements but also increases CPU consumption. |
-|enable_udf                                |  -   |    FALSE     | Whether to enable UDF .                            |
+##### max_allowed_in_element_num_of_delete
+
+- Unit: -
+- Default: 10000
+- Description: The maximum number of elements allowed for the IN predicate in a DELETE statement.
+
+##### enable_materialized_view
+
+- Unit: -
+- Default: TRUE
+- Description: Whether to enable the creation of materialized views.
+
+##### enable_decimal_v3
+
+- Unit: -
+- Default: TRUE
+- Description: Whether to support the DECIMAL V3 data type.
+
+##### enable_sql_blacklist
+
+- Unit: -
+- Default: FALSE
+- Description: Whether to enable blacklist check for SQL queries. When this feature is enabled, queries in the blacklist cannot be executed.
+
+##### dynamic_partition_check_interval_seconds
+
+- Unit: s
+- Default: 600
+- Description: The interval at which new data is checked. If new data is detected, StarRocks automatically creates partitions for the data.
+
+##### dynamic_partition_enable
+
+- Unit: -
+- Default: TRUE
+- Description: Whether to enable the dynamic partitioning feature. When this feature is enabled, StarRocks dynamically creates partitions for new data and automatically deletes expired partitions to ensure the freshness of data.
+
+##### max_partitions_in_one_batch
+
+- Unit: -
+- Default: 4096
+- Description: The maximum number of partitions that can be created when you bulk create partitions.
+
+##### max_query_retry_time
+
+- Unit: -
+- Default: 2
+- Description: The maximum number of query retries on an FE.
+
+##### max_create_table_timeout_second
+
+- Unit: s
+- Default: 600
+- Description: The maximum timeout duration for creating a table, in seconds.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/0118afb4-2152-412d-b42b-60f2fec24ab0)
", in seconds" can be deleted.